oak_functions/loader/src/server.rs

```diff
@@ -714,3 +718,31 @@ pub fn format_bytes(v: &[u8]) -> String {
         .map(|s| s.to_string())
         .unwrap_or_else(|_| format!("{:?}", v))
 }
+
+// The EndPoint of a birectional channel. Sender and Receiver are exposed.
+pub struct EndPoint {
+    pub sender: Sender<AbiMessage>,
+    pub receiver: Receiver<AbiMessage>,
+}
+
+// Create a channel with two symmetrial endpoints.  The AbiMessage sent from one EndPoint are
+// received at the other EndPoint and vice versa by connecting two unidirectional
+// tokio::mpsc channels.
+//
+// In ASCII art:
+//  sender _____  ____ sender
```

Comment:
I would put this in a code block too, so it gets rendered correctly in the documentation.

Reply:
Done.